packages/http-specs/specs/streaming/jsonl/mockapi.ts

```diff
@@ -0,0 +1,39 @@
+import { MockRequest, passOnSuccess, ScenarioMockApi } from "@typespec/spec-api";
+
+export const Scenarios: Record<string, ScenarioMockApi> = {};
+
+Scenarios.Streaming_Jsonl_Basic_send = passOnSuccess({
+  uri: "/streaming/jsonl/send",
+  method: "post",
+  request: {
+    headers: {
+      "Content-Type": "application/jsonl",
+    },
+    body: '{"desc": "one"}\n{"desc": "two"}\n{"desc": "three"}',
```

Comment:
nit: `'{"desc": "one"}\n{"desc": "two"}\n{"desc": "three"}'` appears for 3 times so maybe you could define a constant variable to reuse it.